<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TXT合并器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>获取源</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_src">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>591</width>
       <height>51</height>
      </rect>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>源目录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_bsrc">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>801</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>输出EPUB</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_epubadd">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>591</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_epubb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>591</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>选定保存地址</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_epubrun">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>运行</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_epubname">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>书名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_epubwriter">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>作者</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_epubright">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>版权</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_epubpub">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>出版社</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_epubload">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>80</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>加载预设</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>封面</string>
    </property>
    <widget class="QLabel" name="label_img">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_bimg">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>选择封面</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_loadimg">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>加载预设</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>801</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>输出TXT</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_txtadd">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>591</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_txtb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>591</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>选定保存地址</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_txtrun">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>运行</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_txtname">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>书名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_txtwriter">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>作者</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_txtload">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>80</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>加载预设</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
